slides/r-adoption-jan2023.qmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "MMRM and ASA SWE - R Adoption Series"
+author: "Ben Arancibia and Yoni Sidi"
+format: 
+  revealjs:
+    theme: dark
+    incremental: false
+    logo: https://github.com/RConsortium/asa-biop-swe-wg/raw/main/sticker/sticker-1200.png
+    slide-number: c/t
+title-slide-attributes: 
+  data-background-image: https://github.com/RConsortium/asa-biop-swe-wg/raw/main/sticker/sticker-1200.png
+  data-background-size: 20%
+  data-background-opacity: "0.5"
+  data-background-position: 80% 10%
+editor: visual
+---
+
+## Agenda
+
+-   Introduction and Overview of AWA SWE
+
+-   Mix Models for Repeated Measures - Why is it a Problem?
+
+-   MMRM Package
+
+    -   Why this is not "yet another package"
+
+    -   Long Term Perspective
+
+    -   Comparing MMRM R Package to SAS - Demo
+
+-   Closing and Next Steps
+
+## Software Engineering in Biostatistics
+
+-   Open-source software has gained increasing popularity in Biostatistics over the last two decades
+    -   Pros: rapid uptake of novel statistical methods and unprecedented opportunities for collaboration and innovation
+    -   Cons: users face huge variability in software quality, in particular reliability, efficiency and maintainability
+-   Developing high-quality software with good coding practices, reproducible outputs, and self-sufficient documentation is critical to inform clinical and regulatory decisions
+
+## How to deal with these issues?
+
+-   To deal with the issues of statistical quality assurances for R packages and creating high quality statistical software a group came together to create through the ASA called the "Software Engineering Working Group" SWE WG to look R packages from a statistical point of view
```

Comment:
```suggestion
-   To deal with the issues of statistical quality assurances for R packages and creating high quality statistical software a group came together to create through the "Software Engineering Working Group" (SWE WG) to look R packages from a statistical point of view
```
plus consider breaking down into several bullets / or remove bullet (if it is just one bullet)